app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx

```diff
@@ -149,6 +156,15 @@ export class LogToolbar extends Component {
           allEventClick={LOG_PAGE_EVENTS.ALL_LABEL_BREADCRUMB}
           onRestorePath={restorePath}
         />
+        <InputCheckbox
+          onChange={() => {
+            this.setState({ isChecked: !this.state.isChecked });
```

Comment:
For now, it looks like this checkbox does nothing and will do nothing. Please, rename checkbox state according to its meaning. Also, expose checkbox state and onChange callback if they should be lifted to the parent component.